app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :likes, foreign_key: :author_id
   has_many :comments, foreign_key: :author_id
 
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
```

Comment:
- [ ] Kindly test this function to see if it can return the 3 most recent posts.

Reply:
Hi @fickryiman, this test is not yet added 👍🏿